src/reference/config/bindings/binding-tcp.md

```diff
@@ -15,9 +15,6 @@ Zilla runtime tcp binding.
 tcp_server:
   type: tcp
   kind: server
-  options:
-    host: 0.0.0.0
-    port: 12345
```

Comment:
These don't need to be removed from the global sample unless they are being removed from the spec. For each binding, we want a description and Full example code block. We will update the examples to use the new feature but reference docs can describe everything.